pkg/util/env.go

```diff
@@ -10,6 +10,13 @@ import (
 	"strings"
 )
 
+type ApplyEnvFn func(key string, value string, ok bool) error
+
+func ApplyEnv(key string, consumer ApplyEnvFn) error {
```

Comment:
I know I am not very good at this myself, but for this method I wish we would have a comment, which explains how it works :grinning:

Reply:
Done :)